pkg/apb/watch_pod.go

```diff
@@ -60,3 +68,12 @@ func watchPod(podName string, namespace string) error {
 
 	return fmt.Errorf("Timed out while watching pod %s for completion", podName)
 }
+
+func errorPullingImage(conds []apiv1.PodCondition) bool {
+	for _, cond := range conds {
+		if cond.Reason == "ErrImgPull" {
```

Comment:
I didn't find "ErrImgPull" anywhere in vendor, only "ErrImagePull". Is "ErrImgPull" the error we want or is it coming from something we don't have vendored? https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/images/types.go#L33

Reply:
We are using the pod's [status](https://godoc.org/k8s.io/api/core/v1#PodStatus) field. The [conditions](https://godoc.org/k8s.io/api/core/v1#PodStatus) slice is where I am looking for this value. It appears that the reason is a string and this is set to ErrImgPull which you can see when you get the pod w/ yaml when it can not pull the image. 

Is there a better way to get this? I would prefer a better way.

Reply:
I think we can do something like https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/kubelet_pods.go#L1114-L1147

Reply:
I will re-do this section based on that. It looks much more correct!